import/index_java/src/org/vufind/index/FormatCalculator.java

```diff
@@ -311,7 +316,9 @@ protected String getFormatFromRecordType(Record record, char recordType, Control
                         return "Video";
                     default: break;
                 }
-                return "Slide";
+                // Insufficient info in LDR and 008 to distinguish still from moving images
+                // Leave it to corresponding 007 if it exists, else fall back to "ProjectMedium"  
+                return (formatCodes007.contains('g')) ? "" : "ProjectedMedium";
```

Comment:
Sorry if I'm confused, but is this logic inverted? If the format code indicates that this is a projected graphic, why are we returning empty string, and returning ProjectedMedium for all other format codes?

Reply:
Twisted maybe, but I think it is right. Basically if we know there is a 007 for the same type of thing ("g" = project medium), return "" so that the 007 value is the only one used. If there isn't a 007 for the same type of thing, return "ProjectedMedium" as a best guess. Individual instances can map this to something else (includign the old default 'Slide') based on their content. Does that make sense. If you look back at the original, we were actually returning "" for similar reasons

Reply:
Btw, this currently isn't compiling. I'll take a look later

Reply:
I see, so if we have an 007 value of 'g' we've already assigned something more specific in `getFormatFrom007`, so adding a generic ProjectedMedium value here would be redundant and unhelpful. Maybe it's worth adjusting the comment to say that a little more explicitly to prevent future confusion. And if nothing else, there's a typo of "ProjectMedium" in the current comment.

If you need help with the compile problem, let me know and I can look again tomorrow. When you're ready, I can reindex again and report back with revised numbers!

Reply:
Comments improved and compile problem solved